download.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+set -e
+
+test -z "$VERSION" && {
```

Comment:
Why not put the latest as a fallback for VERSION like the others and update it on every release?

Reply:
+1, if you put the version in this file, then this is the only file that needs to be updated on every version bump. All other uses of `download.sh` don't need to have a version number unless they want to override it

Reply:
`latest` (see [here](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/latest)) is not working right now as we do not have any release (with a green label, right now we have only pre-releases 😉 )

We can do a separate issue for it.

Reply:
I created https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1250

Reply:
I was thinking that in putting `v0.3.1-beta.1` until #1250 is done and update it on every release.